packages/odyssey-react/src/components/TextInput/TextInput.tsx

```diff
@@ -43,7 +44,7 @@ interface CommonProps
    * The underlying input element type
    * @default text
    */
-  type?: "text" | "email" | "url" | "tel" | "password";
+  type?: "text" | "email" | "url" | "tel" | "search" | "password";
```

Comment:
One thought I had on keeping these types exposed is that if someone uses the `PasswordInput` or the `SearchInput` it would be nice if the native types matched so that any default behavior the browser adds for prompting for saved passwords, or for autofilling search values would still be available?
Or maybe we don't want that? Or maybe there is another way to achieve that?
@jeffbelser-okta